packages/ui/src/molecules/QuantitySelector/QuantitySelector.tsx

```diff
@@ -0,0 +1,96 @@
+import type {
+  AriaAttributes,
+  ReactNode,
+  InputHTMLAttributes,
+  MouseEventHandler,
+} from 'react'
+import React from 'react'
+
+import Input from '../../atoms/Input'
+import IconButton from '../IconButton'
+import { PlusIcon, MinusIcon } from './stories/assets/Icons'
+
+export interface QuantitySelectorProps
+  extends InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * Returns the value of element's class content attribute.
+   */
+  className?: string
```

Comment:
I don't think it's necessary to declare className because it already exists in InputHTMLAttributes <HTMLInputElement>